packages/big-design/src/components/Worksheet/editors/ToggleEditor/ToggleEditor.tsx

```diff
@@ -25,29 +24,29 @@ const InternalToggleEditor = ({ rowId, toggle }: ToggleEditorProps) => {
     setEditingCell({ cell: null });
   }, [hasExpanded, isExpandable, onToggle, setEditingCell, toggle]);
 
-  return isExpandable ? (
-    <Flex justifyContent="center">
-      {hasExpanded ? (
-        <StyledExpandButton
-          iconOnly={<ExpandMoreIcon />}
-          onClick={() => {
-            onToggle(true);
-          }}
-          title="toggle row expanded"
-          variant="subtle"
-        />
-      ) : (
-        <StyledExpandButton
-          iconOnly={<ChevronRightIcon />}
-          onClick={() => {
-            onToggle(false);
-          }}
-          title="toggle row expanded"
-          variant="subtle"
-        />
-      )}
-    </Flex>
-  ) : null;
+  if (!isExpandable) {
+    return null;
+  }
+
+  return hasExpanded ? (
```

Comment:
probably if we use another if in here might look cleaner 🍹 
````tsx
if (hasExpanded) {
  return ( 
  <StyledExpandButton
      iconOnly={<ExpandMoreIcon color="secondary60" />}
      onClick={() => {
        onToggle(true);
      }}
      title="toggle row expanded"
      variant="subtle"
    />
  )
}

return (
<StyledExpandButton
      iconOnly={<ChevronRightIcon color="secondary60" />}
      onClick={() => {
        onToggle(false);
      }}
      title="toggle row expanded"
      variant="subtle"
    />
)
```

Reply:
@MariaJose Thanks, done))